index.bs

```diff
@@ -1957,6 +1980,11 @@ To <dfn>obtain a randomized source response pick rate</dfn> given a [=randomized
 1. Let |numPossibleValues| be the [=set/size=] of |possibleValues|.
 1. Return |numPossibleValues| / (|numPossibleValues| - 1 + e<sup>|epsilon|</sup>).
 
+To <dfn>obtain pick rate for a source with attribution scope</dfn> given a positive 32-bit integer |numSourceStates|, a positive 32-bit integer |numEventStates|, a positive 32-bit integer |numUsedScopes|, a positive and a double |epsilon|:
```

Comment:
Why aren't we using this as the pick rate for randomised response when we have attrbution scoping?

Reply:
Synced offline, the noise probability should remain the same.